### PR TITLE
Add 32 bit support

### LINKED ISSUE #219: 32 bit x86 support?
We hear some folks are using 32 bit x86 machines so we should try to support that if we can.

### PR DESCRIPTION
**Pull request by Hoverbear**
##### Description

Closes #219.

TODO:

* [ ] Add buildkite job

##### Checklist

- [x] Formatted with `cargo fmt`
- [x] Built with `nix build`
- [x] Ran flake checks with `nix flake check`
- [ ] Added or updated relevant tests (leave unchecked if not applicable)
- [ ] Added or updated relevant documentation (leave unchecked if not applicable)
- [ ] Linked to related issues (leave unchecked if not applicable)

##### Validating with `install.determinate.systems`

If a maintainer has added the `upload to s3` label to this PR, it will become available for installation via `install.determinate.systems`:

```shell
curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix/pr/$PR_NUMBER | sh -s -- install
```

### CODE REVIEW

**Comment by Hoverbear** [comment]
The latest ubuntu I can find to test this on is https://mirror.it.ubc.ca/ubuntu-releases/16.04/

**Comment by Hoverbear** [comment]
image
It works!

File: src/settings.rs [hunk 1]
```diff
@@ -11,6 +11,9 @@ use crate::channel_value::ChannelValue;
 /// Default [`nix_package_url`](CommonSettings::nix_package_url) for Linux x86_64
 pub const NIX_X64_64_LINUX_URL: &str =
     "https://releases.nixos.org/nix/nix-2.12.0/nix-2.12.0-x86_64-linux.tar.xz";
```
Comment (cole-h) [inline src/settings.rs:13]: Maybe not in this PR but:

Should we be bumping the Nix version? Latest stable is already 2.13.2.

**Review by cole-h — APPROVED** [review]
LGTM overall.

File: src/settings.rs (hunk 1 above)
Comment (Hoverbear) [reply src/settings.rs:13]: Ya! I was bumping it in #196 but that is delayed so I can do that separately.

**Pull request merged**